src/js/control-bar/volume-menu-button.js

```diff
@@ -19,6 +19,12 @@ import VolumeBar from './volume-control/volume-bar.js';
 class VolumeMenuButton extends MenuButton {
 
   constructor(player, options={}){
+    // If an inline volumeMenuButton is used, we should default to using a horizontal
+    // slider for obvious reasons.
+    if (options.inline) {
```

Comment:
Might as well move this inside the `options.vertical === undefined` check, so it's overridable.